satpy/readers/geos_netcdficare.py

```diff
@@ -0,0 +1,769 @@
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Interface to GEOSTATIONNARY L1B NETCDF data from ICARE (Lille).
+
+Introduction
+------------
+
+The ``geos_netcdficare`` reader reads some geostationnary netcdf build by
+Meteo France and stored at Icare.
+
+The brightness tempera ture and albedo are calibrated.
+
+That has been stored by the ICARE Data and Services Center
+Data can be accessed via: http://www.icare.univ-lille1.fr
+
+This reader concerns the following netcdfs :
+
+. msg with a longitude near 0° :
+Mmultic3kmNC4_msg03_20231113_111500.nc
+Mmultic1kmNC4_msg03_20231113_111500.nc
+
+. Msg rapid scan with a longitude near 9.5° :
+Mrsmultic3kmNC4_msg03_20231113_111500.nc
+Mrsmultic1kmNC4_msg03_20231113_111500.nc
+
+. Msg with a longitude near 42° :
+Imultic3kmNC4_msg03_20231113_111500.nc
+Imultic1kmNC4_msg03_20231113_111500.nc
+
+. Himawari :
+Jmultic2kmNC4_hima09_20231113_111500.nc
+Jmultic1kmNC4_hima09_20231113_111500.nc
+Jmultic500mNC4_hima09_20231113_111500.nc
+
+. Goesr near -137° :
+Wmultic2kmNC4_goes16_202406281000.nc.
+The better resolution are not built at Lannion, only at Tahiti.
+
+. Goesr in -75° :
+Emultic2kmNC4_goes16_202406281000.nc
+Emultic1kmNC4_goes16_202406281000.nc
+Emultic500mNC4_goes16_202406281000.nc
+
+. Mtg :
+Mmultic2km_mtgi1_20240104_090000.nc
+Mmultic1km_mtgi1_20240104_090000.nc
+Mmultic500m_mtgi1_20240104_090000.nc
+
+
+Example:
+--------
+Here is an example how to read the data in satpy:
+
+	from satpy import Scene
+	import glob
+
+	filenames = glob.glob('data/*2019-03-01T12-00-00*.hdf')
+	scn = Scene(filenames = filenames, reader = 'hima_netcdficare')
+	scn.load(['true_color'])	# scn.load(['VIS006'])
+
+	my_area = AreaDefinition(
+		'my_area', 'zone', 'my_area',
+		'+proj=latlong +lon_0=0 +a=6378169 +b=6356583 +h=35785831 +x_0=0
+			+y_0=0 +pm=0',
+		8500, 4000,
+		[-180., -80., 180., 80],
+		nprocs=16)
+
+	natscn = scn.resample(my_area, resampler='nearest')
+	natscn.save_dataset(composite_name, filename = filename_image_out)
+
+"""
+
+import datetime as dt
+
+import numpy as np
+
+from satpy.readers._geos_area import get_area_definition, get_area_extent
+
+# from netCDF4 import Dataset
+# from xarray import DataArray
+import xarray as xr
+
+from satpy.readers.file_handlers import BaseFileHandler
+
+# from satpy._compat import cached_property
+# from satpy.utils import get_dask_chunk_size_in_bytes
+import dask
+from satpy.readers import open_file_or_filename
+# import math
+
+
+class NETCDF_ICARE(BaseFileHandler) :
+	# Cf readers/file_handlers.py.
+
+	def __init__(self, filename, filename_info, filetype_info) :
+		"""Init the file handler."""
+
+		super().__init__(filename, filename_info, filetype_info)
+
+		# chunk_bytes = self._chunk_bytes_for_resolution()
+		chunk_bytes = '128 MiB'
+		# chunk_bytes = '64 MiB'
+
+		# with dask.config.set(**{'array.slicing.split_large_chunks': True}) :
+		with dask.config.set({"array.chunk-size": chunk_bytes}) :
```

Comment:
This should be set by the user outside of satpy, so this code needs to be removed.